config/global.ini.php

```diff
@@ -356,6 +356,11 @@
 ; If you need any other period, or want to ensure one of those is always archived, you can define them here
 archiving_custom_ranges[] =
 
+; If configured, archiving queries will be aborted after the configured amount of seconds. Set it to -1 if the query time
+; should not be limited. Note: This feature requires a recent MySQL version (5.7 or newer). Some MySQL forks like MariaDB
+; might not support this feature which uses the MAX_EXECUTION_TIME hint.
+archiving_query_max_execution_time = 7200
```

Comment:
Would it be a good idea to add a test to `DbHelper::addMaxExecutionTimeHintToQuery()` w/ `max execution time == -1`? Currently it's only tested when the value is `0`.